jujugui/static/gui/src/app/components/deployment-flow/credential/add/add.js

```diff
@@ -113,118 +112,106 @@ YUI.add('deployment-credential-add', function() {
       const user = this.props.user;
       this.props.setCredential(`${user}/${templateName}`);
       this.props.setRegion(this._getRegion());
-      this.props.setTemplate(this._generateTemplate());
+      this.props.setTemplate(this._generateCredentials());
       this.props.close();
     },
 
     /**
+      Set the authType state when the select changes.
+
+      @method _handleAuthChange
+      @param {String} authType The selected authType.
```

Comment:
The auth types are things like `userpass` or `access-key` they are provided from the top level objects for the forms e.g. https://github.com/juju/juju-gui/pull/1997/files#diff-d3414ab33b3dbc61a2b3294b2822026fR58